app/javascript/packs/brave_rewards_banner_display.jsx

```diff
@@ -237,8 +278,10 @@ class BraveRewardsPageForm extends React.Component {
     const body = new FormData();
 
     body.append('title', document.getElementsByClassName("sc-gZMcBi")[0].innerText);
-    body.append('description', document.getElementsByClassName("sc-gqjmRU")[0].innerText);
+    body.append('title', document.getElementsByClassName("sc-gZMcBi")[0].innerText);
```

Comment:
I believe you accidentally copied the above line

Reply:
Fixed